packages/api/src/derives/cennzx/assetWithdrawn.ts

```diff
@@ -36,9 +36,9 @@ export function assetToWithdraw(instanceId: string, api: ApiInterfaceRx) {
       map(([tradeAssetReserve, coreAssetReserve, totalLiquidity]) =>
         getAssetToWithdraw(
           new BN(liquidity.toString()),
-          coreAssetReserve as any,
-          tradeAssetReserve as any,
-          totalLiquidity as any
+          new BN(coreAssetReserve.toString()),
```

Comment:
Are toString calls here necessary?

Reply:
yes.. because the coreAssetReserve we receive is of type Balance and we need to string of balance to create BN object..